packages/frontapp/src/app/pages/SearchPage.tsx

```diff
@@ -35,6 +36,12 @@ const Topics = styled.h5`
   font-size: ${dimensions.xl};
 `;
 
+const WrapperLoader = styled.div`
```

Comment:
i think its better to move `WrapperLoader` to the `Loader` Component itself, and import it from there, rather than copy pasting the styles in every file